.travis.yml

```diff
@@ -1,30 +1,45 @@
 language: php
 
-php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - hhvm
+sudo: false
 
-env:
-  - SYMFONY_VERSION=2.1.*
-  - SYMFONY_VERSION=2.2.*
-  - SYMFONY_VERSION=2.3.*
-  - SYMFONY_VERSION=dev-master
+branches:
+  only:
+    - master
+
+matrix:
+  include:
+    - php: 5.3
+      env:
+        - deps=low
+        - SYMFONY_VERSION=2.7.*
+    - php: 5.4
+      env:
+        - deps=high
+        - SYMFONY_VERSION=2.7.*
+    - php: 5.5
+      env:
+        - deps=low
+        - SYMFONY_VERSION=2.8.*
+    - php: 5.6
+      env:
+        - deps=high
+        - SYMFONY_VERSION=3.0.*
+    - php: 7.0
+      env:
+        - deps=low
+        - SYMFONY_VERSION=3.1.*
+    - php: 7.1
+      env:
+        - deps=high
+        - SYMFONY_VERSION=3.1.*
```

Comment:
We know more or less which php versions are bc (so you could remove 5.4, 5.5 and 5.6 for example).
I'm not especially against testing the bundle in many environnements but it slows down the test build and it is not convenient for contributors imo.